contrib/runners/action_chain_runner/action_chain_runner.py

```diff
@@ -506,6 +508,20 @@ def _render_publish_vars(action_node, action_parameters, execution_result,
     @staticmethod
     def _resolve_params(action_node, original_parameters, results, chain_vars, chain_context):
         # setup context with original parameters and the intermediate results.
+        chain_parent = chain_context.get('parent', {})
+        pack = chain_parent.get('pack')
+        user = chain_parent.get('user')
+
+        if pack and user:
+            config_loader = ContentPackConfigLoader(
```

Comment:
Actually, it looks like you probably also wanted to use `_get_config` utility function here?